tasks/in-house-ctf/data/cmd_injection/docker-compose.yaml

```diff
@@ -0,0 +1,23 @@
+services:
```

Comment:
Could we name these files `compose.yaml` (consistent w/ general Docker compose naming standards which is also followed by the Inspect documentation)